help/en/releasenotes/current-draft-note.shtml

```diff
@@ -163,6 +163,7 @@
             <ul>
                 <li>Now sends repeated function commands so long as 
                     the throttle function is active.</li>
+                <li>TMCC2 Legacy 200 speed step mode is now available.</li>
```

Comment:
Should it be TMCC2 and not TMCC? With the digit 2 at the end?

Reply:
Thanks for the careful reading. TMCC2 is the updated protocol with 200 speed steps that’s available in Legacy equipment. Yes, it’s confusing that “Legacy” is the more modern one.